src/SeqViz/SeqViz.tsx

```diff
@@ -6,13 +6,28 @@ import { Annotation, Element, Part } from "../part";
 import { cutSitesInRows } from "../utils/digest";
 import isEqual from "../utils/isEqual";
 import { directionality, dnaComplement } from "../utils/parser";
-import search from "../utils/search";
+import search, { SearchResult } from "../utils/search";
 import { annotationFactory, getSeqType } from "../utils/sequence";
+import { ICutSite } from "./CommonTypes";
 import SeqViewer from "./SeqViewer";
 import CentralIndexContext from "./handlers/centralIndex";
 import { SelectionContext, defaultSelection } from "./handlers/selection";
 import "./style.css";
 
+export interface SeqVizSelection {
```

Comment:
This makes a bit more sense coming from `./handlers/selection`. Analogous to SearchResults in search